config/settings.py

```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'django-insecure-*_g&bnywvu$@12^*6d6qwe#oqur38w_db11=(w%+w)uz*&b^rj'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
App is currently in Development Stage so it will be helpful to spot bugs easily with Debug mode enabled. Could you revert the changes here ?